.vscode/launch.json

```diff
@@ -18,7 +18,7 @@
 			"type": "extensionHost",
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
-			"args": ["${workspaceRoot}/package.json" ,"--extensionDevelopmentPath=${workspaceRoot}", "--extensionTestsPath=${workspaceRoot}/out/test" ],
+			"args": ["${workspaceRoot}/test/resources/sampleNodeApp/", "--disable-extensions", "--extensionDevelopmentPath=${workspaceRoot}", "--extensionTestsPath=${workspaceRoot}/out/test" ],
```

Comment:
Why are you using `"test/resources/sampleNodeApp"` I think we can remove this one

Reply:
@sudhirverma this is used as mentioned by official docs https://code.visualstudio.com/docs/extensions/testing-extensions#_running-tests-automatically-on-travis-ci-build-machines, the reason for doing it is otherwise I need to mock/stub the whole project in the workspace which would be unnecessary. WDYT?